src/Calculator/GraphingSettings.xaml.cpp

```diff
@@ -0,0 +1,220 @@
+#include "pch.h"
+
+#include "GraphingSettings.xaml.h"
+#include <CalcViewModel\Common\AppResourceProvider.cpp>
+
+using namespace Graphing;
+
+using namespace CalculatorApp;
+using namespace CalculatorApp::Controls;
+
+using namespace Platform;
+using namespace Windows::Foundation;
+using namespace Windows::Foundation::Collections;
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Controls;
+using namespace Windows::UI::Xaml::Controls::Primitives;
+using namespace Windows::UI::Xaml::Data;
+using namespace Windows::UI::Xaml::Input;
+using namespace Windows::UI::Xaml::Media;
+using namespace Windows::UI::Xaml::Navigation;
+
+GraphingSettings::GraphingSettings()
+{
+    InitializeComponent();
+
+    auto resLoader = AppResourceProvider::GetInstance();
+
+    x_MinHeading->Text = resLoader.GetResourceString(L"x-min");
+    x_MaxHeading->Text = resLoader.GetResourceString(L"x-max");
+    y_MinHeading->Text = resLoader.GetResourceString(L"y-min");
+    y_MaxHeading->Text = resLoader.GetResourceString(L"y-max");
+
+    GridHeading->Text = resLoader.GetResourceString(L"GridHeading");
+    UnitsHeading->Text = resLoader.GetResourceString(L"UnitsHeading");
+
+    Radians->Content = resLoader.GetResourceString(L"TrigModeRadians");
+    Degrees->Content = resLoader.GetResourceString(L"TrigModeDegrees");
+    Gradians->Content = resLoader.GetResourceString(L"TrigModeGradians");
+}
+
+void GraphingSettings::GraphingCalculator_DataContextChanged(FrameworkElement ^ sender, DataContextChangedEventArgs ^ args)
+{
+    // The graph control we are working with
+    m_ParentGrapher = (GraphControl::Grapher ^) args->NewValue;
+}
+
+void GraphingSettings::TrigUnitModeClick(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
+{
+    if (Degrees->IsChecked->Value == true)
+    {
+        m_ParentGrapher->TrigUnitMode = (int)Graphing::EvalTrigUnitMode::Degrees;
+    }
+
+    if (Radians->IsChecked->Value == true)
+    {
+        m_ParentGrapher->TrigUnitMode = (int)Graphing::EvalTrigUnitMode::Radians;
+    }
+
+    if (Gradians->IsChecked->Value == true)
+    {
+        m_ParentGrapher->TrigUnitMode = (int)Graphing::EvalTrigUnitMode::Grads;
+    }
+}
+
+void GraphingSettings::UpdateExtents(Platform::Object ^ sender)
+{
+    FrameworkElement ^ fe = (FrameworkElement ^) sender;
```

Comment:
nit, here and in the rest of the file we should use static_cast/dynamic_cast instead of c-style casts
```suggestion
    auto ^ fe = static_cast<FrameworkElement ^>(sender);
```

Reply:
Yep, I should have gone back and done these after I learned this lesson in the tracing PR.  Fixed.